### PR TITLE
5-add-mkdocs

### LINKED ISSUE #5: implement mkdocs
To provide the best documentation on earth, we choose the best tool on earth: mkdocs
In this documenation we want to store our learnings, and general documenation. 

**AC:**
* You can run mkdocs locally via container.

### PR DESCRIPTION
**Pull request by LaurinStreng**
Close #5 

## Current State
Adds the following:
- [x] run mkdocs localy
- [x] Description how to run it localy
- [x] MkDocs Structure is created
- [x] Github Action (linting)
- [x] github pages 

## How to review
- Github Pages works see https://eseldevelopment.github.io/ecmanager/
- Linting CI Works
- Local Mkdocs works
- Default MKdocs structure is setup

### CODE REVIEW

File: mkdocs/mkdocs.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+site_name: My site
```
Comment (tzabbi) [inline mkdocs/mkdocs.yml:1]: Do we can implement a nice design and plugins? For now it is basically enough, but I think we should implement, darkmode, nice color scheme, and some necessary plugins like search etc, ...

Here is a nice tutorial: https://www.youtube.com/watch?v=Q-YA_dA8C20&pp=ygUGbWtkb2Nz 

Maybe you found a better one. What do you think?

Comment (LaurinStreng) [reply mkdocs/mkdocs.yml:1]: Yeah thats a nice Idea im adding some default stuff from ([The Docs ](https://squidfunk.github.io/mkdocs-material/setup/)).

File: mkdocs/mkdocs.yml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+site_url: https://mydomain.org
```
Comment (Marcel2603) [inline mkdocs/mkdocs.yml:2]:
```suggestion
site_url: https://eseldevelopment.github.io/ecmanager
```

**Comment by Marcel2603** [comment]
looks good to me :) 

Well done

**Review by Marcel2603 — APPROVED** [review]

**Pull request merged**